src/DebugEngineHost/HostMarshal.cs

```diff
@@ -59,6 +59,16 @@ public static IDebugFunctionPosition2 GetDebugFunctionPositionForIntPtr(IntPtr l
         }
 
         /// <summary>
+        /// Obtain a string from a BSTR
+        /// </summary>
+        /// <param name="stringId"></param>
+        /// <returns></returns>
+        public static string GetStringForIntPtr(IntPtr stringId)
```

Comment:
I think I would suggest calling this something like 'GetDataBreakpointStringForIntPtr' since this shouldn't be used as a general purpose thing.